react/components/EditorContainer/StoreEditor/Store/PWAForm/components/withPWAMutations.tsx

```diff
@@ -0,0 +1,76 @@
+import { path } from 'ramda'
+import React from 'react'
+import { Mutation, MutationFn, MutationResult } from 'react-apollo'
+
+import { handleCornerCases } from '../../utils/utils'
+import UpdateManifest from '../mutations/UpdateManifest.graphql'
+import UpdateManifestIcon from '../mutations/UpdateManifestIcon.graphql'
+import Manifest from '../queries/Manifest.graphql'
+import Styles from '../queries/Styles.graphql'
+
+export interface ManifestMutationData {
```

Comment:
I think `ManifestData` is better

Reply:
What about [this ManifestData](https://github.com/vtex-apps/admin-pages/pull/166/files#diff-aa7ec14c99aba0a8b5b89c74152d27dbR24)?